l1-contracts/contracts/bridge/interfaces/IL1Bridge.sol

```diff
@@ -40,14 +84,35 @@ interface IL1Bridge {
     ) external;
 
     function finalizeWithdrawal(
+        uint256 _chainId,
         uint256 _l2BatchNumber,
         uint256 _l2MessageIndex,
         uint16 _l2TxNumberInBatch,
         bytes calldata _message,
         bytes32[] calldata _merkleProof
     ) external;
 
-    function l2TokenAddress(address _l1Token) external view returns (address);
+    function bridgehubDeposit(
+        uint256 _chainId,
+        address _prevMsgSender,
+        bytes calldata _data
+    )
+        external
+        payable
+        returns (
+            // which is abi encoded :
+            // address _l1Token,
+            // uint256 _amount,
+            // address _l2Receiver
```

Comment:
Old comment? If this is a struct we don't need to explain what field is as it is declared in the struct itself

Reply:
I was referring to the input _data and not the output struct, but the linter moved it.